README.md

```diff
@@ -60,3 +60,40 @@ ogp_custom_meta_tags = [
 ]
 
 ```
+
+## Per Page Overrides
+[Field lists](https://www.sphinx-doc.org/en/master/usage/restructuredtext/field-lists.html) are used to allow you to override certain settings on each page and set unsupported arbitrary OpenGraph tags.
+
+Make sure you place the fields at the very start of the document such that Sphinx will pick them up and also won't build them into the html.
+
+### Overrides
+These are some overrides that can be used, you can actually override any tag and field lists will always take priority.
+
+* `:og_description_length:`
+  * Configure the amount of characters to grab for the description of the page. If the value isn't a number it will fall back to `ogp_description_length`. Note the slightly different syntax because this isn't directly an OpenGraph tag.
+* `:og:description:`
+  * Lets you override the description of the page.
+* `:og:title:`
+  * Lets you override the title of the page.
+* `:og:type:`
+  * Override the type of the page, for the list of available types take a look at https://ogp.me/#types.
+* `:ogp:image:`
+  * Set the image for the page.
+* `:ogp:image:alt:`
+  * Sets the alt text. Will be ignored if there is no image set.
+
+### Example
+Remember that the fields **must** be placed at the very start of the file. You can verify Sphinx has picked up the fields if they aren't shown in the final html file.
+
+```rst
+:og:description: New description
+:og:image: http://example.org/image.png
+:og:image:alt: Example Image
+```
```

Comment:
```suggestion
```rst
:og:description: New description
:og:image: http://example.org/image.png
:og:image:alt: Example Image

Page contents
=============
```
```

The examples should be fleshed out a little to show that the field list is at the top of the file.